contracts/kleros/KlerosLiquid.sol

```diff
@@ -398,7 +399,7 @@ contract KlerosLiquid is TokenController, Arbitrator {
             require(RN != 0, "Random number is not ready yet.");
             phase = Phase.drawing;
         } else if (phase == Phase.drawing) {
-            require(disputesWithoutJurors == 0 || now - lastPhaseChange >= maxDrawingTime, "There are still disputes without jurors and the maximum drawing time has not passed yet.");
```

Comment:
This description is correct. The error message is the negation of the require (what happens when the require is not satisfied).

Reply:
Restored